migrations/versions/a55bb6a85c83_use_uuidmixin.py

```diff
@@ -0,0 +1,54 @@
+"""Use UuidMixin
+
+Revision ID: a55bb6a85c83
+Revises: 15aede1ebe6f
+Create Date: 2017-07-13 14:23:03.655538
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'a55bb6a85c83'
+down_revision = '15aede1ebe6f'
+
+from uuid import uuid4
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+from sqlalchemy_utils import UUIDType
+from progressbar import ProgressBar
```

Comment:
I'm unclear whether to do this since this is just a one-time migration. Hasjob itself doesn't need it.